apps/web/src/components/NavArrow.tsx

```diff
@@ -0,0 +1,43 @@
+import { useRouter } from "next/router";
+import { ChevronLeftIcon, ChevronRightIcon } from "@heroicons/react/24/outline";
+
+export function NavArrows({ next, prev }: { next?: string; prev?: string }) {
+  return (
+    <div className="flex items-center justify-center gap-1">
+      <NavArrow type="Prev" href={prev} />
+      <NavArrow type="Next" href={next} />
+    </div>
+  );
+}
+
+function NavArrow({ type, href }: { type: "Next" | "Prev"; href?: string }) {
+  const router = useRouter();
+
+  return (
+    <button
```

Comment:
Can we use the button component we already have?

Reply:
I'll make that change after the button refactorization PR is merged, since the style of this button is a bit different